### DISCUSSION TITLE
스크린샷(Selfie) 관련 설정 문의

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1266] [5. Q&A]
![zep_Selfie](https://user-images.githubusercontent.com/12184282/188783438-3a2aab9f-1050-48dc-87be-a534ac574892.jpg)

1.  공식 샘플에서 배경만 없앤 후 빌드 실행 하였더니 , 배경 없애기 버튼 실행후 위의 샷처럼 다시 복귀 되지 않네요 .
2.  공식 샘플의 Scene을 Package 로 개인 작업 Project 로 가져 왔는데 위의 현상과 함께 Selfie Stick이 적용 되지 않습니다 .
     Animator IK 설정 은 했습니다 .
* 혹시 배경등에 별도로 Layer 설정을 해 줘야 하는 것인지요 ?
<Error 메시지 입니다 .>
![Error](https://user-images.githubusercontent.com/12184282/189068329-4f5c4406-e823-4586-994e-75279302e371.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 보여주신 에러는 스크린샷 모듈을 가져올 때 제대로 세팅이 되지 않은 것으로 보여집니다.
스크린샷에 필요한 스크립트 및 프리팹은 깃헙에서 가져오신 후 메타파일을 삭제하시기는 것을 추천 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/581#discussioncomment-2506605
https://github.com/naverz/zepeto-studio-kor/discussions/1173

내 프로젝트에 가져온 후에는 아래 가이드에 따라 차근차근 설정을 확인해 보세요.
https://studio.zepeto.me/guides/screenshot

감사합니다.
